tools/extra/fpgadiag/mactest.py

```diff
@@ -30,20 +30,30 @@
 import argparse
 import binascii
 import sys
+import glob
 from common import FpgaFinder, exception_quit
-from common import COMMON, hexint
+from common import COMMON, convert_argument_str2hex
```

Comment:
if it's from the same library, couldn't we import it with one line? (just a style thing)

Reply:
fixed